### DISCUSSION TITLE
grapes with laravel

### DISCUSSION THREAD

**Discussion by mr-masoud-momeni** [GrapesJS/grapesjs#4169] [Q&A]
i implemented grapesjs in laravel. All steps of data storage and retrieval are done in the database. But my goal is to create a dynamic application that has the ability to have different sections of the site on the page. Like showing a slideshow or showing recent posts on the site. What is your suggestion for implementation? Because a part of the page is created that is stored in HTML database. When retrieving from the database, it is fully executed and no request is sent to the server.

**Comment by mr-masoud-momeni** [comment]
Another problem is that JavaScript is not stored in the Grapes code so that I can send a request to the server via JavaScript when page loads.

**Comment by Vac1911** [comment]
I'm using symfony but working with the same idea of storing pages and loading them from the database.
Using remote storage did most of what I needed: https://grapesjs.com/docs/modules/Storage.html#setup-remote-storage
And you can store and load templates: https://grapesjs.com/docs/modules/Storage.html#store-and-load-templates

**Comment by Bishoy-Saber** [comment]
@mr-masoud-momeni 
I am facing a problem while trying to store and loading data in db by APIs . May I ask if can you share the code?
Thank You.
